src/ImmersedBoundaries/ImmersedBoundaries.jl

```diff
@@ -105,24 +105,33 @@ struct ImmersedBoundaryGrid{FT, TX, TY, TZ, G, I, M, Arch} <: AbstractGrid{FT, T
     architecture :: Arch
     underlying_grid :: G
     immersed_boundary :: I
-    active_cells_map :: M
+    active_cells_interior :: M
```

Comment:
```suggestion
    interior_active_cells :: M
```

Though I have to admit, I don't know what this object means. I just know that I would never use the phrase "active cells interior" in a sentence...